packages/wpcom-template-parts/src/utils.ts

```diff
@@ -1,24 +1,30 @@
 import { translate as translateMethod } from 'i18n-calypso';
 
 export const getAutomatticBrandingNoun = ( translate: typeof translateMethod ) => {
+	const AutomatticBrand = 'Automattic';
+	const options = {
+		description:
+			'Branding to be shown on the Footer of the page, Automattic\'s variable will always contains the word "Automattic"',
+		args: { Automattic: AutomatticBrand },
+	};
 	const automatticRoger = [
-		translate( 'An Automattic brainchild' ),
-		translate( 'An Automattic contraption' ),
-		translate( 'An Automattic creation' ),
-		translate( 'An Automattic experiment' ),
-		translate( 'An Automattic invention' ),
-		translate( 'An Automattic joint' ),
-		translate( 'An Automattic medley' ),
-		translate( 'An Automattic opus' ),
-		translate( 'An Automattic production' ),
-		translate( 'An Automattic ruckus' ),
-		translate( 'An Automattic thingamajig' ),
+		translate( 'An %(Automattic)s brainchild', options ),
+		translate( 'An %(Automattic)s contraption', options ),
+		translate( 'An %(Automattic)s creation', options ),
+		translate( 'An %(Automattic)s experiment', options ),
+		translate( 'An %(Automattic)s invention', options ),
+		translate( 'An %(Automattic)s joint', options ),
+		translate( 'An %(Automattic)s medley', options ),
+		translate( 'An %(Automattic)s opus', options ),
+		translate( 'An %(Automattic)s production', options ),
+		translate( 'An %(Automattic)s ruckus', options ),
+		translate( 'An %(Automattic)s thingamajig', options ),
 	];
 
-	const branding = automatticRoger[ Math.floor( Math.random() * ( 10 - 0 + 1 ) + 0 ) ];
+	const branding = automatticRoger[ Math.floor( Math.random() * ( 10 - 0 + 1 ) + 0 ) ]?.toString();
```

Comment:
I'm wondering what these `±0` arithmetics are supposed to achieve. To select a random array element, seems to me that simple `roger[ Math.floor( Math.random() * roger.length ) ]` is sufficient.

Also, once you avoid the `.split` call and use interpolation to insert the A8c logo, the `.toString` call is no longer needed. In theory `branding` can be a HTML/JSX markup then.

Reply:
> I'm wondering what these ±0 arithmetics are supposed to achieve. To select a random array element, seems to me that simple roger[ Math.floor( Math.random() * roger.length ) ] is sufficient.

Agreed, I'm also not sure why are they needed, but I suppose it must be needed as it is a non-trivial addition.